ui/ducks/send/send.test.js

```diff
@@ -485,7 +485,7 @@ describe('Send Slice', () => {
         expect(result.recipient.error).toStrictEqual('invalidAddressRecipient');
       });
 
-      // TODO: This seems off, is this suppose to be a networks that dont implement checksum? But the user input is not valid hex in the first place.
+      // TODO: Expectation might change in the future
```

Comment:
Leaving this as a TODO since there is still refactors to the error handling on user input, is that ok?

Reply:
Yeah, that's fine!